capa/render/result_document.py

```diff
@@ -540,3 +582,39 @@ def from_capa(cls, meta, rules: RuleSet, capabilities: MatchResults) -> "ResultD
             )
 
         return ResultDocument(meta=Metadata.from_capa(meta), rules=rule_matches)
+
+    @classmethod
+    def parse_raw(cls, raw: str, rules: RuleSet):
```

Comment:
since this returns something that isn't just a `ResultDocument`, lets move this routine to a helper function outside of the `ResultDocument` scope. actually, lets just rely on the existing `ResultDocument.parse_raw/parse_file` for this.

then, lets move the logic that converts a `ResultDocument` instance into the metadata/capabilities pair into its own method on `ResultDocument`. i think this should probably be called `ResultDocument.to_capa() -> Tuple[Dict, Dict]` so that we can invoke it on any ResultDocument instance.